lib/mp-readline/readline.c

```diff
@@ -98,6 +98,72 @@ typedef struct _readline_t {
 
 STATIC readline_t rl;
 
+#if MICROPY_REPL_EMACS_WORDS_MOVE
+STATIC size_t cursor_count_forward_word(size_t cursor_pos, vstr_t *line) {
+    size_t pos = rl.cursor_pos;
+    const char *line_buf = vstr_str(line);
+
+    if (pos == vstr_len(rl.line)) {
+        return pos;
+    }
+
+    assert(pos < vstr_len(rl.line));
+    if (!unichar_isalnum(line_buf[pos])) {
+        pos++;
+
+        while (pos < vstr_len(rl.line)) {
+            if (unichar_isalnum(line_buf[pos])) {
+                break;
+            }
+            pos++;
+        }
+    }
+
+    if (pos == rl.line->len) {
```

Comment:
It';s confusing to sometimes use `->len` and other times `vstr_len`

Reply:
Yeah it's my miss. Will update to `vstr_len`.